http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20220626_add_disk_source.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="leonardo" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet logicalFilePath="leonardo" author="dvoet" id="add_source_disk">
+        <addColumn tableName="PERSISTENT_DISK">
+            <column name="sourceDisk" type="varchar(1024)">
```

Comment:
I added this to the database for consistency and so that round-tripping a `PersistentDisk` object to the database would not lose information. I don't think the feature requires this though. Comment if you think it should be removed.